sklearn/cluster/_bisect_k_means.py

```diff
@@ -189,6 +190,16 @@ class BisectKMeans(KMeans):
         Sum of squared distances of samples to their closest cluster center,
         weighted by the sample weights if provided.
 
+    n_features_in_ : int
+        Number of features seen during :term:`fit`.
+
+        .. versionadded:: 0.24
```

Comment:
Because the entire algorithm is new, I don't think you need to add these `versionadded` flags.

Reply:
Good point